packages/flutter_tools/lib/src/test/flutter_platform.dart

```diff
@@ -306,6 +310,7 @@ class FlutterPlatform extends PlatformPlugin {
   final FlutterProject? flutterProject;
   final String? icudtlPath;
   final TestTimeRecorder? testTimeRecorder;
+  final UriConverter? uriConverter;
```

Comment:
Could be worth adding a doc comment to explain the use case for passing a non-null uriConverter